### ISSUE TITLE
Making an elementary tutorial to get started with each commands

### ISSUE THREAD

**Issue by Hearen** [alibaba/arthas#210]
There are user cases as: https://github.com/alibaba/arthas/issues?q=label%3Auser-case

But it's kind of not **elementary** enough for beginners to get the quick overview of Arthas. 

So I have been thinking about to draft up a tutorial to help beginners to quickly understand:

- **what** Arthas can do?
- **how** to use Arthas?

If this sounds good to you, please let me know and then I will start on it. 

Basically I will try to start based on #178 and make it more readable and understandable via consistent demo. 

@ralf0131 cc @hengyunabc

**Comment by ralf0131** [comment]
Looks good to me! Please go ahead!

**Comment by Hearen** [comment]
#223
